internal/runbits/requirements/requirements.go

```diff
@@ -209,6 +222,61 @@ func (r *RequirementOperation) ExecuteRequirementOperation(
 		pg = nil
 	}
 
+	if r.Auth.Authenticated() && operation == bpModel.OperationAdded && ns.Type() == model.NamespacePackage {
```

Comment:
What about bundles that pull in individual packages? Are they checked too?

Reply:
I don't believe bundles were considered here... I'm not sure how much we are pushing their use. Looking at the dashboard you have to go to the project configuration page to see the CVEs for each package in the bundle rather than a report for the bundle itself. 

I'm also pretty sure that bundles are just for Perl so I'm not sure how much we want to complicate this by dealing with bundles.